spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/controller/StreamController.java

```diff
@@ -200,18 +205,28 @@ public void deploy(@PathVariable("name") String name, @RequestParam(required = f
 			Exception {
 		StreamDefinition stream = this.repository.findOne(name);
 		Assert.notNull(stream, String.format("no stream defined: %s", name));
-		deployStream(stream);
+		deployStream(stream, DeploymentPropertiesUtils.parse(properties));
 	}
 
-	private void deployStream(StreamDefinition stream) {
+	private void deployStream(StreamDefinition stream, Map<String, String> deploymentProperties) {
+		if (deploymentProperties == null) {
+			deploymentProperties = Collections.emptyMap();
+		}
 		Iterator<ModuleDefinition> iterator = stream.getDeploymentOrderIterator();
 		for (int i = 0; iterator.hasNext(); i++) {
 			ModuleDefinition module = iterator.next();
 			String type = (i == 0) ? "sink"
 					: (iterator.hasNext() ? "processor" : "source");
 			ModuleCoordinates coordinates = this.registry.findByNameAndType(module.getName(), type);
-			// todo: pass deployment properties
-			this.deployer.deploy(new ModuleDeploymentRequest(module, coordinates));
+			Map<String, String> moduleDeploymentProperties = new HashMap<>();
+			String modulePrefix = String.format("module.%s.", module.getLabel());
```

Comment:
I hope the prefix `module.*.` that applies the property to all modules isn't the scope of this PR.